packages/mwp-app-render/src/components/__snapshots__/connectWithMatchMedia.test.jsx.snap

```diff
@@ -1,60 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`connectWithMatchMedia exists 1`] = `
-ShallowWrapper {
-  "length": 1,
-  Symbol(enzyme.__root__): [Circular],
-  Symbol(enzyme.__unrendered__): <Connect(ConnectWithMatchMedia(TestComponent)) />,
-  Symbol(enzyme.__renderer__): Object {
-    "batchedUpdates": [Function],
-    "getNode": [Function],
-    "render": [Function],
-    "simulateEvent": [Function],
-    "unmount": [Function],
-  },
-  Symbol(enzyme.__node__): Object {
-    "instance": null,
-    "key": undefined,
-    "nodeType": "function",
-    "props": Object {
-      "dispatch": [Function],
-      "media": Object {
-        "isAtSmallUp": true,
-      },
-    },
-    "ref": null,
-    "rendered": null,
-    "type": [Function],
-  },
-  Symbol(enzyme.__nodes__): Array [
+<ConnectWithMatchMedia(TestComponent)
```

Comment:
Why were all these other snapshots affected?

Reply:
This repo wasn't using the 'pretty' enzyme snapshot serializer that mup-web is using, so the snapshots were accurate but not very legible - `snapshotSerializer` added [here](https://github.com/meetup/meetup-web-platform/pull/429/files#diff-04e7f2718acefccd3077079b98af4b79R16)